src/index.html

```diff
@@ -22,6 +22,405 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="container">
+      <a
+        href="/"
+        class="logo_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__links">
+          <li>
+            <a
+              href="http://"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards cards-container--position">
+      <section
+        data-qa="card"
+        class="card"
+      >
+        <div class="card__product-img">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K (MNED2UA/A)"
+            class="card__img"
+          />
+        </div>
+
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <section class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </section>
+          <p class="card__stars-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-txt">Price:</p>
+          <p class="card__price-sum">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button-buy"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </section>
```

Comment:
This section of the card is repeated multiple times in the HTML file. Consider using a templating engine or JavaScript to generate these sections dynamically. This will help to keep your code DRY (Don't Repeat Yourself) and make it easier to maintain.